src/views/LabView.vue

```diff
@@ -187,11 +188,11 @@ export default Vue.extend({
             return this.$store.state.current_lab.lab.title;
         },
         descriptiontoShow(): string{
-            // strip all html, only take the first MAX_CHARS characters and add ending ... tag
-            return this.$store.state.current_lab.lab.body.replace(/<[^>]+>/ig,"").substr(0, MAX_CHARS) + ". . .";
+            // Limit to MAX_CHARS and add ellipsis
+            return DOMPurify.sanitize(`${this.$store.state.current_lab.lab.body.substr(0, MAX_CHARS)}...`)
```

Comment:
This could cause broken HTML. Can we rely on CSS for elipsis?

Reply:
Happily. Just matching what was there before.